.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Ah, i think this may not have been intended? If you can remove this we can merge! Thanks!!!